pandas/core/arrays/categorical.py

```diff
@@ -1847,6 +1847,66 @@ def __iter__(self):
         """Returns an Iterator over the values of this Categorical."""
         return iter(self.get_values().tolist())
 
+    @staticmethod
+    def _contains(key, categories, container):
+        """Helper for membership check for ``key``.
+
+        This helper method is used in :method:`Categorical.__contains__`
+        and in :class:`CategoricalIndex.__contains__`.
+
+        Returns True if ``key`` is in ``categories`` and the
+        location of ``key`` in ``categories`` is in ``container``.
+
+        Notes
+        -----
+        This method does not check for Nan values. Do that separately
+        before calling this method.
```

Comment:
Nice!  Couple of things:

* Notes section goes at the bottom
* The first line ("Helper for membership check...") should be underneath the triple quotes.

Reply:
Ok, both done.

But I've never noticed "the first line should be underneath the triple quotes"-rule and it's not widely applied in pandas. Are you sure that's a style rule?